### ISSUE TITLE
The Intermediate file is almost blank

### ISSUE THREAD

**Issue by ilovefuju** [tomguluson92/StyleGAN_PyTorch1.0.1#9]
Hi,
Thanks a lot for your pytorch version code.
I am training my own dataset with this model. But, the intermediate file till 80 epoch is still almost blank. 
since my dataset has a lot of white background images, I am  wondering if it is the main reason.
![63](https://user-images.githubusercontent.com/53335269/66265460-666a2600-e849-11e9-8090-d887c3ec2dec.png)

**Comment by tomguluson92** [comment]
Hi, you can check #4

**Comment by meghanakanapaneni** [comment]
> Hi,
> Thanks a lot for your pytorch version code.
> I am training my own dataset with this model. But, the intermediate file till 80 epoch is still almost blank.
> since my dataset has a lot of white background images, I am wondering if it is the main reason.
> ![63](https://user-images.githubusercontent.com/53335269/66265460-666a2600-e849-11e9-8090-d887c3ec2dec.png)

Hi, I am also struck with the same issue. Did you solve this issue? Please help me if you have solved. Thanks in advance.

**Comment by RuoyuChen10** [comment]
> > Hi,
> > Thanks a lot for your pytorch version code.
> > I am training my own dataset with this model. But, the intermediate file till 80 epoch is still almost blank.
> > since my dataset has a lot of white background images, I am wondering if it is the main reason.
> > ![63](https://user-images.githubusercontent.com/53335269/66265460-666a2600-e849-11e9-8090-d887c3ec2dec.png)
> 
> Hi, I am also struck with the same issue. Did you solve this issue? Please help me if you have solved. Thanks in advance.

maybe cause of normalization for the input of training set, try remove it?.

**Comment by meghanakanapaneni** [comment]
Thank you I will try
